### DISCUSSION TITLE
Deploy record stuck

### DISCUSSION THREAD

**Discussion by ramazulay** [D10S0VSkY-OSS/Stack-Lifecycle-Deployment#254] [Show and tell]
Hi @D10S0VSkY-OSS, 
How are you?
I installed the latest version and it looks amazing, thanks for your hard work.
I have a some bugs there, can I make an appointment and I will show you?

**Comment by D10S0VSkY-OSS** [comment]
Hi @ramazulay 
Fixed ⬆ Hotfix locked task when worker crash issue #257 
Regards
